DashAI/back/dataloaders/classes/dataloader.py

```diff
@@ -162,4 +166,37 @@ def split_dataset(
         dataset["train"] = train_split["train"]
         dataset["test"] = test_valid_split["train"]
         dataset["validation"] = test_valid_split["test"]
+
+        train_dataset_dict = dataset["train"].to_dict()
+        test_dataset_dict = dataset["test"].to_dict()
+        validation_dataset_dict = dataset["validation"].to_dict()
+
+        train_dataset = Dataset.from_dict(train_dataset_dict)
+        test_dataset = Dataset.from_dict(test_dataset_dict)
+        validation_dataset = Dataset.from_dict(validation_dataset_dict)
+
+        separate_dataset_dict = DatasetDict(
+            {
+                "train": train_dataset,
+                "test": test_dataset,
+                "validation": validation_dataset,
+            }
+        )
+
+        dataset = self.to_dataset_dashai(
+            separate_dataset_dict, inputs_columns, outputs_columns
+        )
+        return dataset
+
+    @staticmethod
+    def to_dataset_dashai(
```

Comment:
Just a small detail, but I think it is more common to use the package name before the function name: `to_dashai_dataset`.

```suggestion
    def to_dashai_dataset(
```